src/lightningrpc.rs

```diff
@@ -19,11 +19,11 @@ pub struct LightningRPC {
 
 /// Optional arguments for pay() request
 #[derive(Debug, Clone, Default)]
-pub struct PayOptions {
+pub struct PayOptions<'f> {
```

Comment:
nit: Why `'f` instead of the common `'a`?

Reply:
I have no clue. I think when I first learned rust I saw this somewhere (always beginning at `'f` and then counting up `'g`, `'h`, ...) and till you mentioned it I was totally sure that this was some kind of standard. But looking it up, it's actually standard to start from `'a`, well, gonna do some `s/'f/'a/g`.